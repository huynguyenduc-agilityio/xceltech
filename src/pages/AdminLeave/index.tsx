const AdminLeave = () => {
  return (
    <div className="px-12 bg-primary rounded-3xl mt-10 flex justify-between max-h-[422px] overflow-hidden">
      <div className="w-[740px]">
        <img
          src="/assets/images/arrowRight.webp"
          alt="Arrow Right Image"
          className="ml-5 object-cover"
        />

        <p className="text-5xl font-bold text-white">
          Manage ALL <span className="text-secondary">Leave Applications</span>
        </p>
        <p className="text-white text-xl">
          A relaxed employee is a performing employee.
        </p>

        <div className="w-full flex justify-end">
          <img
            src="/assets/images/arrowLeft.webp"
            alt="Arrow Left Image"
            className="object-cover"
          />
        </div>
      </div>

      <img
        src="/assets/images/leaveImage.webp"
        alt="Leave Image"
        className="object-cover"
      />
    </div>
  );
};

export default AdminLeave;
