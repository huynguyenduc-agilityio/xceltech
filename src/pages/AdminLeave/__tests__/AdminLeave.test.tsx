import { render } from '@testing-library/react';

// Pages
import AdminLeave from '..';

describe('AdminLeave', () => {
  it('should render successfully', () => {
    const { container } = render(<AdminLeave />);

    expect(container).toMatchSnapshot();
  });
});
