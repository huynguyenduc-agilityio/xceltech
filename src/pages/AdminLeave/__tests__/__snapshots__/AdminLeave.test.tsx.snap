// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminLeave should render successfully 1`] = `
<div>
  <div
    class="px-12 bg-primary rounded-3xl mt-10 flex justify-between max-h-[422px] overflow-hidden"
  >
    <div
      class="w-[740px]"
    >
      <img
        alt="Arrow Right Image"
        class="ml-5 object-cover"
        src="/assets/images/arrowRight.webp"
      />
      <p
        class="text-5xl font-bold text-white"
      >
        Manage ALL 
        <span
          class="text-secondary"
        >
          Leave Applications
        </span>
      </p>
      <p
        class="text-white text-xl"
      >
        A relaxed employee is a performing employee.
      </p>
      <div
        class="w-full flex justify-end"
      >
        <img
          alt="Arrow Left Image"
          class="object-cover"
          src="/assets/images/arrowLeft.webp"
        />
      </div>
    </div>
    <img
      alt="Leave Image"
      class="object-cover"
      src="/assets/images/leaveImage.webp"
    />
  </div>
</div>
`;
